README.md

```diff
@@ -9,11 +9,11 @@ A command-line tool for managing common tasks in Ethereum 2.
 
 ## Table of Contents
 
-- [Install](#install)
-- [Usage](#usage)
-- [Maintainers](#maintainers)
-- [Contribute](#contribute)
-- [License](#license)
+- [Commands](#commands)
+- [HOWTO](#howto)
```

Comment:
Please could you change these back?  They should link to the sections directly in this document.

Reply:
This is an autogenerated TOC from my editor. Seems like it detects the nesting of the headers and formats the TOC according to that. I will disable it